src/main/java/jade/imtp/leap/Stub.java

```diff
@@ -53,7 +53,7 @@
 	protected static final String DISP_ERROR_MSG = "Dispatcher error";
 	
 	// The addresses of the remote object
-	protected List              remoteTAs = new ArrayList();
+	protected ArrayList              remoteTAs = new ArrayList();
```

Comment:
These are jade lists. They do not support the equals method :-( 
To get access to the inner java list I need to toList() method which is not defined for the raw List. Completely stupid!